src/database/migrations/20220920054800-create-launches.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable("launches", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      user_id: {
+        allowNull: false,
+        type: Sequelize.INTEGER,
+      },
+      site_id: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+      },
+      primary_domain_source: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      primary_domain_target: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      domain_validation_source: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      domain_validation_target: {
+        allowNull: false,
+        type: Sequelize.STRING,
+      },
+      created_at: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+      updated_at: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+    })
```

Comment:
I think this is missing a `deleted_at`? The `paranoid` flag is set to `true` in the model.

Reply:
Whoops, this was done in #558, can ignore!